pkg/pillar/cmd/nim/nim.go

```diff
@@ -1057,6 +1057,9 @@ func (n *nim) includeLastResortPort(ifAttrs netmonitor.IfAttrs) bool {
 	exclude := strings.HasPrefix(ifName, "vif") ||
 		strings.HasPrefix(ifName, "nbu") ||
 		strings.HasPrefix(ifName, "nbo") ||
+		strings.HasPrefix(ifName, "can") ||
+		strings.HasPrefix(ifName, "vcan") ||
+		strings.HasPrefix(ifName, "wlan") ||
```

Comment:
@milan-zededa has wlan been causing any issues with lastresort?
True that we don't have the credentials when we do lastresort (and EVE doesn't support connecting to WiFi networks without authentication), but I don't recall seeing any issues for WiFi.

Reply:
Including wlan leads to error `link not up for interface wlan0 (down)` being reported for the interface. This is not a functional issue, but perhaps some users could get confused as to why EVE is trying to use WiFi at all.